paddle/majel/ddim/ddim.cc

```diff
@@ -1,4 +1,4 @@
-#include "paddle/majel/ddim.h"
+#include "paddle/majel/ddim/ddim.h"
```

Comment:
I tend to think that we don't need to move each C++ module into a directory.  In C++, a module is defined by a .cc file, optionally with interface declaration in a .h file.  We can describe a C++ module by a CMake rule, e.g., `cc_library(ddim SRCS ddim.cc)`. So it seems overkilling to move each C++ module into a directory.